src/SFA.DAS.Assessor.Functions/OpportunityFinderDataSync.cs

```diff
@@ -16,7 +16,7 @@ public OpportunityFinderDataSync(IAssessorServiceApiClient assessorServiceApiCli
         }
 
         [FunctionName("OpportunityFinderDataSync")]
-        public async Task Run([TimerTrigger("0 0 7 * * *", RunOnStartup = true)]TimerInfo myTimer, ILogger log)
+        public async Task Run([TimerTrigger("0 0 7 * * *", RunOnStartup = false)]TimerInfo myTimer, ILogger log)
```

Comment:
This should still be true

Reply:
That shouldn't have been checked in.  Good spot!